src/app/components/Figure/Caption/index.jsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import { FF_NEWS_SANS_REG } from '../../../lib/constants/styles';
+
+const Caption = styled.figcaption`
+  background-color: #d5d0cd;
```

Comment:
Opening an issue to confirm this value with UX and add it to the constant style values and then reuse it against the caption and image component.

Reply:
Opened issue https://github.com/bbc/simorgh/issues/322 to address this